packages/shared/src/types/cloudFunctions.ts

```diff
@@ -27,8 +27,6 @@ export type CreateAuthUserPayload = Partial<{
 export interface QueryAuthStatusPayload {
   /** Current organization */
   organization: string;
-  /** String used to authenticate user, either email or phone */
-  authString: string;
```

Comment:
We should probably [also change the invocation](https://github.com/eisbuk/EisBuk/blob/711f4120955f0677096a8b1957f3fee3cf9f3510/packages/client/src/store/actions/authOperations.ts#L67) of this cloud function to omit `authString` for consistency.